frontend/common/src/components/PageHeader/PageHeader.tsx

```diff
@@ -3,11 +3,42 @@ import Heading from "../Heading";
 
 export interface PageHeaderProps {
   icon?: React.FC<{ className: string }>;
+  subtitle?: string;
```

Comment:
In order to allow for `react-intl` messages this should be the type `React.ReactNode`

```suggestion
  subtitle?: React.ReactNode;
```

Reply:
Changed with [Changed subtitle in PageHeader to be ReactNode](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3956/commits/7c43873984aef83e6a92a796b6b62b574513c43c)